clang/include/clang/Driver/Options.td

```diff
@@ -4979,6 +4979,8 @@ def cheerp_no_math_fround : Flag<["-"], "cheerp-no-math-fround">, Flags<[NoXarch
   HelpText<"Disable JavaScript Math.fround">;
 def cheerp_no_credits : Flag<["-"], "cheerp-no-credits">, Flags<[NoXarchOption]>,
   HelpText<"Disable Cheerp credits in JS">;
+def cheerp_preserve_free : Flag<["-"], "cheerp-preserve-free">, Flags<[NoXarchOption]>,
```

Comment:
I don't think we need a driver option. the only use case is disabling for asan.  There is no point in having this option be user-facing